src/execution/vm/vm.cpp

```diff
@@ -802,7 +802,7 @@ void VM::Interpret(const uint8_t *ip, Frame *frame) {
 
   OP(InitInteger) : {
     auto *sql_int = frame->LocalAt<sql::Integer *>(READ_LOCAL_ID());
-    auto val = frame->LocalAt<int32_t>(READ_LOCAL_ID());
+    auto val = frame->LocalAt<int64_t>(READ_LOCAL_ID());
```

Comment:
@amlatyrngom @pmenon  This is to fix how BIGINT was being handled. Previously it would end up inserting and reading a 0.